login.html

```diff
@@ -0,0 +1,182 @@
+                        data-bs-dismiss="modal"
+                        aria-label="Close"
+                      ></button>
+                    </div>
+                    <div class="modal-body">
+                      <input
+                        placeholder="Write a post and find a partner for bike tour"
+                        aria-label="text"
+                      />
+                    </div>
+                    <div class="modal-footer">
+                      <button
+                        type="button"
+                        class="cta_model"
+                        data-bs-dismiss="modal"
+                      >
+                        Close
+                      </button>
+                      <button type="button" class="cta_model">Post</button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <!-- model finish -->
+
+              <form
+                class="d-flex align-items-center justify-content-center"
+                role="search"
+              >
+                <input
+                  class="form-control me-2"
+                  type="search"
+                  placeholder="Search"
+                  aria-label="Search"
+                />
+                <button class="cta_search" type="submit">Search</button>
+              </form>
+            </div>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="px-4 py-5 my-5 text-center">
+        <img
+          class="login_pic"
+          src="/image/pictures/pic-9.jpg"
+          alt=" two girls are posing to the camera  with there electric bikes"
+        />
+        <div class="col-lg-6 mx-auto">
+          <p class="lead mb-4">
+            we are building bike social media website, this website is for
+            making contacts and bike ride tours, and be friends. please write to
+            use if there is something we can do better for community bike a
+            social media website and has many good review, please keep
+            standards.
+          </p>
+        </div>
+      </div>
+      <div class="container">
+        <div class="row">
+          <div class="col-8 mx-auto">
+            <form action="/profile.html" method="get">
+              <div class="item_one">
+                <label for="email">Email address</label>
+                <input
+                  type="email"
+                  class="form-control"
+                  id="Emailinput"
+                  placeholder="name@example.com"
+                  required
+                />
+              </div>
+              <div class="password">
+                <label for="floatingPassword">Password</label>
+                <input
+                  type="password"
+                  id="Passwordinput"
+                  class="form-control"
+                  placeholder="write your password"
+                  maxlength="10"
```

Comment:
```suggestion
                  minlength="8"
```

It's considered bad form to set a max length for passwords, instead there should be a min length. Check out this article about the subject: https://specopssoft.com/blog/password-length-best-practices

Reply:
i was thinking it about it but i got confuse by that why min length and why max length so i chose choose max length, thank i will go to that steps